src/GridCal/Engine/Simulations/ShortCircuitStudies/short_circuit_driver.py

```diff
@@ -62,10 +62,7 @@ def __init__(self, bus_index=None, fault_type='3x', branch_index=None, branch_fa
         else:
             self.bus_index = bus_index
 
-        if fault_type is None:
-            self.fault_type = list()
-        else:
-            self.fault_type = fault_type
+        self.fault_type = fault_type
```

Comment:
ok, it is almost always advisable to use an enum for anything that is a defined list of options like in this case

Reply:
Not sure how to do that or if it is already fine. I have added you as a collaborator in case you want to improve this

Reply:
should be ok as it is now